rcmd/install.go

```diff
@@ -498,9 +514,58 @@ func InstallPackagePlan(
 			log.Errorf("did not install %s", pkg)
 		}
 	}
+
 	if anyFailed {
 		log.Errorf("installation failed for packages: %s", strings.Join(failedPkgs, ", "))
 		return fmt.Errorf("failed installation for packages: %s", strings.Join(failedPkgs, ", "))
 	}
 	return nil
 }
+
+func writeDescriptionInfo(ir InstallRequest) {
+	_, err := updateDescriptionInfo(
+		filepath.Join(ir.InstallArgs.Library, ir.Package, "DESCRIPTION"),
```

Comment:
I believe this library actually gets updated during the install, when installing through src. We need the final Library where it was ultimately installed. Packages installed from src go through src --> install in tmpdir to binary --> binary copied to binary cache --> installed through binary cache.


as of now all packages installed through src error with:

```
WARN[0038] DESCRIPTION file error:open /var/folders/g_/0948z6350756j3yly26_zqnr0000gn/T/GMWJGSAPGATL/devtools/DESCRIPTION: no such file or directory
```